resources/RemoveInactiveUsersRunner.php

```diff
@@ -57,36 +71,60 @@
     if ($lastLoginDate) { // null lastLoginDate check
         $interval = $today->diff($lastLoginDate);
     } else { // This might only be run once, since new users always have field filled.
-        echo "User has no lastLoginDate (it may have been a very long time.)\n";
-        echo "Deleting user.\n";
+        echo "Deleting this user as it has no last login date " .
+        "(it may have been a very long time). \n";
+        if ($dryRun == true) {  // If dry run option has been selected        
```

Comment:
```suggestion
        if ($dryRun == true) {  // If dry run option has been selected
```

Reply:
Some errant whitespace. You can accept my suggestion to fix it without needed to edit it on your development host.

Reply:
ok